commit ff1c02dd1d72198d1de9197fb1ced09821367f65
AuthorDate: 2025-12-01 20:22:22 +0100
CommitDate: 2026-03-30 15:00:06 +0200
feat(indépendant): réforme des barèmes de cotisation (cas général hors PLR)


diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/conjoint-collaborateur.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/conjoint-collaborateur.publicodes"
index 982e4811f..960fa09e4 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/conjoint-collaborateur.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/conjoint-collaborateur.publicodes"
@@ -90,7 +90,7 @@ indépendant . conjoint collaborateur . cotisations:
     - retraite de base
     - retraite complémentaire
     - invalidité et décès
-    - indemnités journalières maladie
+    - indemnités journalières
 
   avec:
     assiette:
@@ -112,7 +112,7 @@ indépendant . conjoint collaborateur . cotisations:
               - cotisations et contributions . assiette sociale
               - 1 / 3
 
-    indemnités journalières maladie:
+    indemnités journalières:
       produit:
         - unité: €/an
           valeur: 40% * plafond sécurité sociale
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/CSG-CRDS.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/CSG-CRDS.publicodes"
index edb987faf..d2dc9b10b 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/CSG-CRDS.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/CSG-CRDS.publicodes"
@@ -22,6 +22,7 @@ indépendant . cotisations et contributions . CSG-CRDS:
         - revenus de remplacement
 
     déductible . revenus de remplacement:
+      non applicable si: après réforme
       produit:
         - IJSS . montant
         - 3.8%
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/assiette-minimale.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/assiette-minimale.publicodes"
index 60038e979..7080ab293 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/assiette-minimale.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/assiette-minimale.publicodes"
@@ -29,29 +29,20 @@ indépendant . assiette minimale . maladie:
   références:
     Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-cotisations/cotisations-minimales/
 
-indépendant . assiette minimale . invalidité et décès:
-  description: |
-    Si le revenu du chef d’entreprise est déficitaire ou inférieur aux bases de
-    calcul, certaines cotisations seront portées à un montant minimum.
+indépendant . assiette minimale . indemnités journalières:
+  description: La cotisation indemnités journalières n’est pas proratisée.
   produit:
     - plafond sécurité sociale
-    - 11.5%
+    - 40%
   unité: €/an
   arrondi: oui
   références:
     Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-cotisations/cotisations-minimales/
-    Article L632-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000037951125
-    Article D632-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000041966755
 
 indépendant . assiette minimale . retraite:
   description: La cotisation minimale de retraite de base permet de valider 3
     trimestres de retraite, quel que soit le revenu.
   variations:
-    - si: date < 01/2023
-      alors:
-        produit:
-          - plafond sécurité sociale
-          - 11.5%
     - si: entreprise . durée d'activité cette année < 90 jour
       alors: 0
     - sinon: 450 heure/an * SMIC . horaire . début d'année
@@ -60,19 +51,15 @@ indépendant . assiette minimale . retraite:
   références:
     Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-cotisations/cotisations-minimales/
 
-indépendant . assiette minimale . retraite . en 2022:
+indépendant . assiette minimale . invalidité et décès:
   description: |
-    L’assiette minimale de 11,50 % n’était pas suffisante pour valider 3 trimestres.
-    Par conséquent cette assiette minimale a été révisée en cours d’année et sera
-    appliquée de manière rétroactive à compter du 1er janvier 2022.
-
-    L’assiette minimale est de 4 758 € pour 2022 uniquement. Depuis 2023, l’assiette
-    minimale est repassée à 11,5 % du PASS.
-  remplace:
-    références à: assiette minimale . retraite
-    sauf dans: cotisations et contributions . cotisations . invalidité et décès
-  applicable si:
-    toutes ces conditions:
-      - date >= 01/2022
-      - date < 01/2023
-  valeur: 4758 €/an
+    Si le revenu du chef d’entreprise est déficitaire ou inférieur aux bases de
+    calcul, certaines cotisations seront portées à un montant minimum.
+  produit:
+    - PSS proratisé
+    - 11.5%
+  unité: €/an
+  arrondi: oui
+  références:
+    Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-cotisations/cotisations-minimales/
+    Article D632-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000041966755
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes"
index a21e73d6c..abf42479e 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes"
@@ -6,11 +6,11 @@ indépendant . cotisations et contributions:
 
   somme:
     - cotisations et contributions . cotisations
-    - conjoint collaborateur . cotisations
-    - cotisations facultatives . montant
     - CSG-CRDS
-    - contributions spéciales
     - formation professionnelle
+    - cotisations facultatives . montant
+    - conjoint collaborateur . cotisations
+    - contributions spéciales
   note: |
     À la différence des cotisations, les contributions ne sont pas réintroduites
     pour le calcul de la CSG/CRDS. Elles ne bénéficient pas non plus de la
@@ -53,6 +53,8 @@ indépendant . cotisations et contributions . formation professionnelle:
   références:
     Contribution à la formation professionnelle (CFP): https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-formation-professionnelle
     Contribution à la formation professionnelle (CFP) des entrepreneurs individuels: https://entreprendre.service-public.fr/vosdroits/F23459
+    Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-des-cotisations/cotisations-minimales/
+    Article L6331-48 du Code du travail: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000044056633
 
 indépendant . cotisations et contributions . après réforme:
   valeur: date > 01/2025
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes"
index e446d194f..85b71aa81 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes"
@@ -1,13 +1,11 @@
 indépendant . cotisations et contributions . cotisations:
   somme:
     - maladie
-    - indemnités journalières maladie
-    - maladie domiciliation fiscale étranger
+    - allocations familiales
     - retraite de base
     - retraite complémentaire
     - PCV
     - invalidité et décès
-    - allocations familiales
     - (- exonérations . ZFU)
   références:
     Taux de cotisations - Artisan, commerçant et profession libérale non réglementée: https://www.urssaf.fr/accueil/outils-documentation/taux-baremes/taux-cotisations-ac-plnr.html
@@ -17,71 +15,106 @@ indépendant . cotisations et contributions . cotisations:
 
 indépendant . cotisations et contributions . cotisations . maladie:
   description: |
-    Cette cotisation permet de bénéficier de l’ensemble des prestations*
-    maladie-maternité géré par l’assurance maladie et les Cpam :
-
-    - une couverture maladie pour l’assuré et les membres de sa famille, identique
-    à celle des salariés (mêmes taux et mêmes conditions de remboursement) ;
-    - une assurance maternité complète en cas de grossesse ou d’adoption ;
-    - une complémentaire santé solidaire en cas de revenus faibles ou une aide pour
-    souscrire une assurance maladie complémentaire.
-  non applicable si: situation personnelle . domiciliation fiscale à l'étranger
-  valeur: total
-  abattement: indemnités journalières maladie
+    Le montant de la cotisation pour la couverture des risques d’assurance
+    maladie et maternité contient deux composantes :
+    - la cotisation maladie 1 (assurance maladie et maternité)
+    - la cotisation maladie 2 (indemnités journalières maladie)
+  variations:
+    - si:
+        est applicable: domiciliation fiscale étranger
+      alors: domiciliation fiscale étranger
+    - sinon:
+        barème:
+          assiette:
+            valeur: assiette sociale
+            plancher: plancher assiette
+          multiplicateur: PSS proratisé
+          tranches:
+            - taux: taux 1
+              plafond: plafond taux 1
+            - taux: taux 2
   arrondi: oui
   unité: €/an
   références:
-    Article D621-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000046714747
     Cotisations maladie: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-cotisations-maladie
+    Article D621-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904610
+    Article D621-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904592
 
   avec:
-    total:
-      description: |
-        Le montant de la cotisation pour la couverture des risques d’assurance
-        maladie et maternité. Contient deux composantes :
-        - la cotisation maladie 1 (assurance maladie et maternité)
-        - la cotisation maladie 2 (indemnités journalières maladie)
-      note:
-        Pour calculer le montant de la cotisation maladie 1, on soustrait le
-        montant de la cotisation maladie 2 du montant total de la cotisation maladie.
-      barème:
-        assiette:
-          valeur: assiette sociale
-          plancher: assiette minimale . maladie
-        multiplicateur:
-          produit:
-            - plafond sécurité sociale
-            - entreprise . prorata temporis
-        tranches:
-          - taux: taux progressif
-            plafond: 5
-          - taux: 6.50%
-      références:
-        Article D621-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000046714760
+    plancher assiette:
+      variations:
+        - si: après réforme
+          alors: 0
+        - sinon: assiette minimale . maladie
+
+    plafond taux 1:
+      variations:
+        - si: après réforme
+          alors: 3
+        - sinon: 5
 
-    taux progressif:
+    taux 1:
       arrondi: 2 décimales
-      taux progressif:
-        assiette: assiette
-        multiplicateur:
-          produit:
-            - plafond sécurité sociale
-            - entreprise . prorata temporis
-        tranches:
-          - taux: 0%
-            plafond: 40%
-          - taux: 4%
-            plafond: 60%
-          - taux: 6.70%
-            plafond: 110%
-          - taux: 6.70%
-            plafond: 500%
+      variations:
+        - si: après réforme
+          alors:
+            taux progressif:
+              assiette: assiette sociale
+              multiplicateur: PSS proratisé
+              tranches:
+                - taux: 0%
+                  plafond: 20%
+                - taux: 1.50%
+                  plafond: 40%
+                - taux: 4.00%
+                  plafond: 60%
+                - taux: 6.50%
+                  plafond: 110%
+                - taux: 7.70%
+                  plafond: 200%
+                - taux: 8.50%
+                  plafond: 300%
+        - sinon:
+            taux progressif:
+              assiette: assiette sociale
+              multiplicateur: PSS proratisé
+              tranches:
+                - taux: 0%
+                  plafond: 40%
+                - taux: 4%
+                  plafond: 60%
+                - taux: 6.70%
+                  plafond: 110%
+                - taux: 6.70%
+                  plafond: 500%
       références:
-        Article D621-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000046714747/
+        Article D621-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904592
 
-    taux progressif . assiette: assiette sociale
+    taux 2:
+      valeur: 6.50%
+      références:
+        Article D621-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904610
+
+indépendant . cotisations et contributions . cotisations . maladie . assurance maladie maternité:
+  description: |
+    Cette cotisation permet de bénéficier de l’ensemble des prestations
+    maladie-maternité gérées par l’assurance maladie et les Cpam :
+
+    - une couverture maladie pour l’assuré/assurée et les membres de sa famille, identique
+    à celle des salariés (mêmes taux et mêmes conditions de remboursement) ;
+    - une assurance maternité complète en cas de grossesse ou d’adoption ;
+    - une complémentaire santé solidaire en cas de revenus faibles ou une aide pour
+    souscrire une assurance maladie complémentaire.
+  note: |
+    Pour calculer le montant de la cotisation maladie 1, on soustrait le
+    montant de la cotisation maladie 2 du montant total de la cotisation maladie.
+  non applicable si: situation personnelle . domiciliation fiscale à l'étranger
+  valeur: maladie
+  abattement: indemnités journalières
+  arrondi: oui
+  unité: €/an
 
-indépendant . cotisations et contributions . cotisations . indemnités journalières maladie:
+indépendant . cotisations et contributions . cotisations . maladie . indemnités journalières:
   description: |
     Cette cotisation permet d’obtenir le versement d’indemnités journalières en cas
     d’arrêt de travail pour cause de maladie ou d’accident.
@@ -92,7 +125,6 @@ indépendant . cotisations et contributions . cotisations . indemnités journali
   produit:
     - assiette
     - taux
-  arrondi: oui
   plafond:
     non applicable si: date >= 01/2024
     # Le taux indemnité est de 0.5%, dans la loi.
@@ -102,38 +134,77 @@ indépendant . cotisations et contributions . cotisations . indemnités journali
     #
     # Pour les revenu ou le taux de la cotisation total (maladie 1 + 2) est inférieur à 0.85, on
     # plafonne donc la cotis maladie 2 à ce total
-    valeur: maladie . total
+    valeur: maladie
+  arrondi: oui
   unité: €/an
   références:
+    Cotisations maladie: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-cotisations-maladie
     Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-des-cotisations/cotisations-minimales/
-    Quelles sont vos cotisations ?: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html
+    Article D621-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904537
 
   avec:
     assiette:
       valeur: assiette sociale
-      plancher: assiette minimale . maladie
+      plancher: assiette minimale . indemnités journalières
       plafond: 5 * plafond sécurité sociale
 
     taux:
       valeur:
         variations:
           - si: date >= 01/2024
-            alors: 0.5%
+            alors: 0.50%
           - sinon: 0.85%
 
-indépendant . cotisations et contributions . cotisations . maladie domiciliation fiscale étranger:
-  applicable si: situation personnelle . domiciliation fiscale à l'étranger
-  titre: Maladie (domiciliation fiscale à l’étranger)
-  description:
+indépendant . cotisations et contributions . cotisations . maladie . domiciliation fiscale étranger:
+  description: |
     En contrepartie de l’exonération de CSG, les cotisants ont un taux
     maladie plus elevé. Contrairement aux autres assurés commerçants/artisans
     ils ne bénéficient pas de la réduction du taux de la cotisation maladie en
     fonction du revenu déclaré.
+  applicable si: situation personnelle . domiciliation fiscale à l'étranger
+  non applicable si: PL . PAMC
   produit:
     - assiette sociale
     - 14.5%
+  arrondi: oui
   unité: €/an
+
+indépendant . cotisations et contributions . cotisations . allocations familiales:
+  description: |
+    Cette cotisation permet de bénéficier des allocations familiales versées par la
+    caisse d’allocations familiales dans les mêmes conditions que les salariés :
+
+    - compensation des charges familiales (naissance, enfant à charge, garde,
+    rentrée scolaire) ;
+    - prestations liées au logement, handicap, précarité (RSA...).
+
+    Le versement de la plupart de ces prestations est soumis à des conditions de
+    ressources.
+    Le montant des allocations attribuées à partir du 2ème enfant varie en fonction
+    des revenus de la famille.
+  produit:
+    - assiette sociale
+    - taux
   arrondi: oui
+  unité: €/an
+  références:
+    Allocations familiales: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-allocations-familiales
+    Article D613-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000041966324
+
+  avec:
+    taux:
+      arrondi: 2 décimales
+      taux progressif:
+        assiette: assiette sociale
+        multiplicateur:
+          produit:
+            - plafond sécurité sociale
+            - entreprise . prorata temporis
+        tranches:
+          - plafond: 110%
+            taux: 0%
+          - plafond: 140%
+            taux: 3.10%
 
 indépendant . cotisations et contributions . cotisations . retraite de base:
   description: |
@@ -151,16 +222,32 @@ indépendant . cotisations et contributions . cotisations . retraite de base:
       valeur: assiette sociale
       plancher: assiette minimale . retraite
     tranches:
-      - taux: taux
-        plafond: plafond sécurité sociale * entreprise . prorata temporis
-      - taux: 0.6%
+      - taux: taux tranche 1
+        plafond: PSS proratisé
+      - taux: taux tranche 2
   arrondi: oui
+  unité: €/an
   références:
-    Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-des-cotisations/cotisations-minimales/
     Retraite de base: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-retraite-de-base
+    Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-des-cotisations/cotisations-minimales/
+    Article D633-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000048838330
+    Article D633-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904636/2024-07-07
 
   avec:
-    taux: 17.75%
+    taux tranche 1:
+      somme:
+        - 17.15%
+        - taux tranche 2
+      références:
+        Article D633-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904636/2024-07-07
+
+    taux tranche 2:
+      variations:
+        - si: après réforme
+          alors: 0.72%
+        - sinon: 0.60%
+      références:
+        Article D633-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904636/2024-07-07
 
 indépendant . cotisations et contributions . cotisations . retraite complémentaire:
   description: |
@@ -175,19 +262,39 @@ indépendant . cotisations et contributions . cotisations . retraite complément
   barème:
     assiette: assiette sociale
     tranches:
-      - taux: 7%
-        plafond: plafond
-      - taux: 8%
-        plafond: 4 * plafond sécurité sociale
+      - taux: taux tranche 1
+        plafond:
+          variations:
+            - si: après réforme
+              alors: PSS proratisé
+            - sinon: PRCI
+      - taux: taux tranche 2
+        plafond: 4 * PSS proratisé
   arrondi: oui
   unité: €/an
   références:
     Retraite complémentaire: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-retraite-complementaire
+    Article D635-7 du Code de la sécurité sociale: www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000041966498
 
   avec:
-    plafond:
-      acronyme: PRCI
-      titre global: plafond retraite complémentaire des indépendants
+    taux tranche 1:
+      variations:
+        - si: après réforme
+          alors: 8.1%
+        - sinon: 7%
+      références:
+        Article D635-7 du Code de la sécurité sociale: www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000041966498
+
+    taux tranche 2:
+      variations:
+        - si: après réforme
+          alors: 9.1%
+        - sinon: 8%
+      références:
+        Article D635-7 du Code de la sécurité sociale: www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000041966498
+
+    PRCI:
+      titre: plafond retraite complémentaire des indépendants
       valeur:
         produit:
           - entreprise . prorata temporis
@@ -199,22 +306,8 @@ indépendant . cotisations et contributions . cotisations . retraite complément
               - si: date >= 01/2023
                 alors: 40784 €/an
               - sinon: 38916 €/an
-    références:
-      Plafond de cotisations - Régime complémentaire des indépendants: https://legislation.lassuranceretraite.fr/#/bareme?file_leaf_ref=ti_cotisation_plafond_cotisation_rci.aspx
-
-indépendant . cotisations et contributions . cotisations . PCV:
-  titre: Prestations complémentaires vieillesse
-  acronyme: PCV
-  non applicable si: oui
-  description: |
-    Certaines catégories professionnelles bénéficient de
-    prestations complémentaires vieillesse (PCV), auparavant nommées « avantage
-    social vieillesse » (ASV). Cela concerne les médecins généralistes, les
-    chirurgiens-dentistes, les sages-femmes, les auxiliaires médicaux et les
-    directeurs de laboratoires. Ce régime résulte de la prise en charge
-    partielle par l’Assurance maladie de leurs cotisations d’assurance
-    vieillesse sous réserve qu’ils aient exercé leur activité dans le cadre
-    conventionnel.
+      références:
+        Plafond de cotisations - Régime complémentaire des indépendants: https://legislation.lassuranceretraite.fr/#/bareme?file_leaf_ref=ti_cotisation_plafond_cotisation_rci.aspx
 
 indépendant . cotisations et contributions . cotisations . invalidité et décès:
   description: |
@@ -229,56 +322,27 @@ indépendant . cotisations et contributions . cotisations . invalidité et déc
   unité: €/an
   arrondi: oui
   références:
-    Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-des-cotisations/cotisations-minimales/
     Invalidité-décès: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-invalidite-deces
-    Article L632-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000037951125
+    Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-des-cotisations/cotisations-minimales/
     Article D632-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000041966755
     Article D632-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000036471466
 
   avec:
     assiette:
-      experimental: oui
       valeur: assiette sociale
-      plancher:
-        produit:
-          - assiette minimale . invalidité et décès
-          - entreprise . prorata temporis
-      plafond:
-        produit:
-          - plafond sécurité sociale
-          - entreprise . prorata temporis
+      plancher: assiette minimale . invalidité et décès
+      plafond: PSS proratisé
 
-indépendant . cotisations et contributions . cotisations . allocations familiales:
+indépendant . cotisations et contributions . cotisations . PCV:
+  titre: Prestations complémentaires vieillesse
+  acronyme: PCV
+  non applicable si: oui
   description: |
-    Cette cotisation permet de bénéficier des allocations familiales versées par la
-    caisse d’allocations familiales dans les mêmes conditions que les salariés :
-
-    - compensation des charges familiales (naissance, enfant à charge, garde,
-    rentrée scolaire) ;
-    - prestations liées au logement, handicap, précarité (RSA...).
-
-    Le versement de la plupart de ces prestations est soumis à des conditions de
-    ressources.
-    Le montant des allocations attribuées à partir du 2ème enfant varie en fonction
-    des revenus de la famille.
-  produit:
-    - assiette sociale
-    - taux
-  arrondi: oui
-  unité: €/an
-  références:
-    Allocations familiales: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-allocations-familiales
-
-  avec:
-    taux:
-      taux progressif:
-        assiette: assiette sociale
-        multiplicateur:
-          produit:
-            - plafond sécurité sociale
-            - entreprise . prorata temporis
-        tranches:
-          - plafond: 110%
-            taux: 0%
-          - plafond: 140%
-            taux: 3.1%
+    Certaines catégories professionnelles bénéficient de
+    prestations complémentaires vieillesse (PCV), auparavant nommées « avantage
+    social vieillesse » (ASV). Cela concerne les médecins généralistes, les
+    chirurgiens-dentistes, les sages-femmes, les auxiliaires médicaux et les
+    directeurs de laboratoires. Ce régime résulte de la prise en charge
+    partielle par l’Assurance maladie de leurs cotisations d’assurance
+    vieillesse sous réserve qu’ils aient exercé leur activité dans le cadre
+    conventionnel.
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/d\303\251but-activit\303\251.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/d\303\251but-activit\303\251.publicodes"
index 814f1ffef..af2645e1a 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/d\303\251but-activit\303\251.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/d\303\251but-activit\303\251.publicodes"
@@ -28,7 +28,7 @@ indépendant . cotisations et contributions . début activité:
     assiette sociale: assiette forfaitaire
     assiette CSG-CRDS: assiette forfaitaire
     cotisations facultatives: non
-    cotisations . maladie . taux progressif . assiette: assiette forfaitaire maladie
+    cotisations . maladie . indemnités journalières . assiette: assiette forfaitaire indemnités journalières
 
   références:
     'Indépendant : Comment sont calculées vos cotisations ?': https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/comprendre-vos-cotisations.html#ancre-le-calcul-vos-cotisations
@@ -43,7 +43,9 @@ indépendant . cotisations et contributions . début activité:
       références:
         'Indépendant : Vos cotisations en début d’activité': https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/comprendre-vos-cotisations.html#titre-bonasavoir-dc7c46c0-20f6-4d0c-bb9d-d963a1afff95
 
-    assiette forfaitaire maladie:
-      valeur: 40% * plafond sécurité sociale
-      unité: €/an
+    assiette forfaitaire indemnités journalières:
+      produit:
+        - plafond sécurité sociale
+        - 40%
       arrondi: oui
+      unité: €/an
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/acre.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/acre.publicodes"
index 4168a8a7b..e789fae4b 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/acre.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/acre.publicodes"
@@ -36,16 +36,16 @@ indépendant . cotisations et contributions . cotisations . exonérations . ACRE
       abattement: taux * prorata sur l'année
       arrondi: oui
 
-    indemnités journalières maladie:
+    indemnités journalières:
       # On préfère l'exonération pension invalidité si elle est plus avantageuse
       non applicable si: exonérations . pension invalidité . prorata >= prorata sur l'année
       remplace:
-        références à: cotisations . indemnités journalières maladie
+        références à: cotisations . maladie . indemnités journalières
         priorité: 10
         sauf dans:
           - cotisations . maladie
-          - cotisations . indemnités journalières maladie . assiette
-      valeur: cotisations . indemnités journalières maladie
+          - cotisations . maladie . indemnités journalières . assiette
+      valeur: cotisations . maladie . indemnités journalières
       abattement: taux * prorata sur l'année
       arrondi: oui
 
@@ -66,8 +66,8 @@ indépendant . cotisations et contributions . cotisations . exonérations . ACRE
         références à: cotisations . maladie
         priorité: 10
         sauf dans:
-          - cotisations . maladie . total
-          - cotisations . maladie . taux progressif
+          - cotisations . maladie
+          - cotisations . maladie . taux 1
       valeur: cotisations . maladie
       abattement: taux * prorata sur l'année
       arrondi: oui
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/pension-invalidit\303\251.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/pension-invalidit\303\251.publicodes"
index 8c65f2025..48c661d1d 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/pension-invalidit\303\251.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/exon\303\251rations/pension-invalidit\303\251.publicodes"
@@ -49,15 +49,15 @@ indépendant . cotisations et contributions . cotisations . exonérations . pens
       plancher: 1 mois
       unité: mois
 
-    indemnités journalières maladie:
+    indemnités journalières:
       arrondi: oui
       non applicable si: prorata < ACRE . prorata sur l'année
-      valeur: indemnités journalières maladie
+      valeur: maladie . indemnités journalières
       remplace:
-        références à: indemnités journalières maladie
+        références à: maladie . indemnités journalières
         sauf dans:
           - cotisations et contributions . cotisations . maladie
-          - cotisations et contributions . cotisations . indemnités journalières maladie .
+          - cotisations et contributions . cotisations . maladie . indemnités journalières .
             assiette
       abattement: prorata
 
@@ -67,8 +67,8 @@ indépendant . cotisations et contributions . cotisations . exonérations . pens
       remplace:
         références à: maladie
         sauf dans:
-          - cotisations et contributions . cotisations . maladie . total
-          - cotisations et contributions . cotisations . maladie . taux progressif
+          - cotisations et contributions . cotisations . maladie
+          - cotisations et contributions . cotisations . maladie . taux 1
       valeur: maladie
       abattement: prorata
 
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes"
index 2f803bcdf..865504823 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes"
@@ -72,17 +72,17 @@ indépendant . PL . CNAVPL . remplacement retraite:
       valeur: retraite
 
     cotisations et contributions retraite de base taux:
-      remplace: cotisations et contributions . cotisations . retraite de base . taux
+      remplace: cotisations et contributions . cotisations . retraite de base . taux tranche 1
       valeur: retraite . tranche T1 . taux
 
-indépendant . PL . CNAVPL . indemnités journalières maladie:
+indépendant . PL . CNAVPL . indemnités journalières:
   description: |
     Depuis le 1er juillet 2021 les affiliés à l’une des caisses de retraite de
     la CNAVPL peuvent bénéficier des indemnités journalières lors d’un arrêt
     maladie.
 
     En conséquence une nouvelle cotisation est créée.
-  remplace: cotisations et contributions . cotisations . indemnités journalières maladie
+  remplace: cotisations et contributions . cotisations . maladie . indemnités journalières
   non applicable si: PL . CNBF
   produit:
     - assiette
@@ -184,9 +184,9 @@ indépendant . PL . CNAVPL . conjoint collaborateur . retraite:
       valeur: conjoint collaborateur . assiette
       plancher: assiette minimale . retraite
 
-indépendant . PL . CNAVPL . conjoint collaborateur . indemnités journalières maladie:
+indépendant . PL . CNAVPL . conjoint collaborateur . indemnités journalières:
   applicable si: date >= 01/2022
-  remplace: conjoint collaborateur . cotisations . indemnités journalières maladie
+  remplace: conjoint collaborateur . cotisations . indemnités journalières
   produit:
     - 40% * plafond sécurité sociale
     - 0.3%
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNBF.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNBF.publicodes"
index fa7851892..7dd4a336e 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNBF.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNBF.publicodes"
@@ -5,7 +5,7 @@ indépendant . PL . CNBF:
     La Caisse Nationale des Barreaux Français (CNBF) est l’organisme de sécurité
     sociale des avocats.
   rend non applicable:
-    - cotisations et contributions . cotisations . indemnités journalières maladie
+    - cotisations et contributions . cotisations . maladie . indemnités journalières
     # Applicable mais pas encore supporté
     - conjoint collaborateur
   références:
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/PAMC.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/PAMC.publicodes"
index ac4eda620..65f9c07db 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/PAMC.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/PAMC.publicodes"
@@ -11,7 +11,7 @@ indépendant . PL . PAMC:
   rend non applicable:
     - conjoint collaborateur
     - revenus étrangers
-    - cotisations et contributions . cotisations . maladie domiciliation fiscale étranger
+    - cotisations et contributions . cotisations . maladie . domiciliation fiscale étranger
     - assiette minimale . maladie
 
   formule: oui
diff --git "a/modele-ti/r\303\250gles/protection-sociale.publicodes" "b/modele-ti/r\303\250gles/protection-sociale.publicodes"
index dee119cdf..9cfbc45bd 100644
--- "a/modele-ti/r\303\250gles/protection-sociale.publicodes"
+++ "b/modele-ti/r\303\250gles/protection-sociale.publicodes"
@@ -120,7 +120,7 @@ protection sociale . retraite . base . cotisée:
   valeur:
     indépendant . cotisations et contributions . cotisations . retraite de
     base / indépendant . cotisations et contributions . cotisations . retraite
-    de base . taux
+    de base . taux tranche 1
   plafond: plafond sécurité sociale
   description: |
     Le revenu cotisé pour la retraite de base est égal à :
@@ -135,10 +135,9 @@ protection sociale . retraite . base . cotisée:
 
     Dans tous les cas, le résultat est limité au plafond de la sécurité sociale.
   références:
-    Article R351-9 du code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028751530/2014-03-21
+    Article R351-9 du code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028751530
     Article R634-1 du code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000038787378
-    Article R551-9 du code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028751530/
-    Guide CNAVPL: https://www.cnavpl.fr/wp-content/uploads/2022/03/guideweb-2022.pdf#page=26
+    Guide de l’assurance vieillesse des professions libérales (2025): https://www.cnavpl.fr/wp-content/uploads/2025/07/Guide-2025.pdf
   note: |-
     Rappel : la validation de trimestres s’effectue, pour les travailleurs
     indépendants à partir du **revenu cotisé**
@@ -399,7 +398,7 @@ protection sociale . maladie:
   avec:
     raam:
       titre: Revenu d’activité annuel moyen
-      valeur: indépendant . cotisations et contributions . cotisations . indemnités journalières maladie . assiette
+      valeur: indépendant . cotisations et contributions . cotisations . maladie . indemnités journalières . assiette
       plafond:
         variations:
           - si: entreprise . activité . libérale . réglementée
